CHANGELOG.md

```diff
@@ -4,7 +4,13 @@
 [Full Changelog](https://github.com/jborgers/sonar-pmd/compare/4.0.3..master)
 
 **Implemented highlights**
-* [TODO]
+* Generate rules xml for the plugin directly from the PMD 7.15.0 rules descriptions: makes all Java and Kotlin rules available and up-to-date automatically
```

Comment:
Generate rules xml for the plugin --> Generate Sonar rules xml
PMD 7.15.0 rules descriptions --> PMD 7.15.0 rules xml
Kotlin not yet, right?

Reply:
The Kotlin rules xml is also generated. There are only two Kotlin rules in PMD as of now. Let me check if these Kotlin rules (still) work in current release.